packages/atomic/src/components/facets/atomic-facet/atomic-facet.tsx

```diff
@@ -178,6 +178,28 @@ export class AtomicFacet implements InitializableComponent, BaseFacet<Facet> {
    */
   @MapProp() @Prop() public dependsOn: Record<string, string> = {};
 
+  /**
+   * Specifies an explicit list of `allowedValues` in the Search API request, separated by commas.
+   *
+   * If you specify a list of values for this option, the facet uses only these values (if they are available in
+   * the current result set).
+   *
+   * Example:
+   *
+   * The following facet only uses the `Contact`, `Account`, and `File` values of the `objecttype` field. Even if the
+   * current result set contains other `objecttype` values, such as `Message`, or `Product`, the facet does not use
+   * those other values.
+   *
+   * ```html
+   * <atomic-facet field="objecttype" allowed-values="Contact,Account,File"></div>
+   * ```
+   *
+   * The maximum amount of allowed values is 25.
+   *
+   * Default value is `undefined`, and the facet uses all available values for its `field` in the current result set.
+   */
+  @Prop() public allowedValues?: string;
```

Comment:
I feel like the markup in other frameworks like React, Angular and Vue might be a bit weird if we take a comma-separated string instead of an array of string.

Maybe something we could do is something similar to `MapProp`. `Prop()` would generate an array of string in the doc, and `ArrayProp` would split & trim the attribute on commas.

I'm not certain this would work though, since MapProp is basically making other attributes to the one matching the prop, and this would be mapping the same attribute to itself.